com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -354,12 +360,16 @@ void InitializeEnvironment()
                     UnityEngine.Random.InitState(unityRlInitParameters.seed);
                     // We might have inference-only Agents, so set the seed for them too.
                     m_InferenceSeed = unityRlInitParameters.seed;
+                    Capabilities = unityRlInitParameters.TrainerCapabilities;
+                    Capabilities.WarnOnPythonMissingBaseRLCapabilities();
                 }
-                catch
+                catch (Exception e)
                 {
                     Debug.Log($"" +
                         $"Couldn't connect to trainer on port {port} using API version {k_ApiVersion}. " +
-                        "Will perform inference instead."
+                        "Will perform inference instead." +
+                        "Original Exception Message:" +
```

Comment:
I think this shows up in the console whenever inference is run - it's already a bit confusing to users, and I think adding the exception will make it worse.

Reply:
ah, yeah.  That's true.  I'll revisit this because i've run into lots of trouble debugging in this tiny bit of code. Logged as MLA-948.